hockeysdk/src/main/java/net/hockeyapp/android/CrashManager.java

```diff
@@ -403,64 +403,72 @@ public static synchronized void submitStackTraces(final WeakReference<Context> w
     }
 
     private static void submitStackTrace(final WeakReference<Context> weakContext, String filename, CrashManagerListener listener, CrashMetaData crashMetaData) {
+        String stacktrace = null;
+        try {
+            stacktrace = contentsOfFile(weakContext, filename);
+        } catch (Exception e) {
+            HockeyLog.error("Failed to read crash data", e);
+        }
+        if (stacktrace.length() == 0) {
+            HockeyLog.warn("The crash data is invalid");
+            deleteStackTrace(weakContext, filename);
+            if (listener != null) {
+                listener.onCrashesSent();
```

Comment:
Why is that not `onNoCrashesFound` since we are not sending anything?

Reply:
Ah because we already onCrashesFound earlier. Well it's weird then, maybe we should just call `onCrashesNotSent` then?